.travis.yml

```diff
@@ -98,6 +98,8 @@ install:
 script: |
   set -e
   if [[ $JOB = docs ]]; then
+    # install docs dependencies
+    pip install '.[docs]'
```

Comment:
oh i was refering to the other `pip install` in  https://github.com/pazz/alot/blob/master/.travis.yml#L77  this can be replaced? The current patch just adds another `pip install` command